scripts/run.py

```diff
@@ -547,7 +543,7 @@ def emission_capacity(clim_emissions, monthly_traffic):
     emission_cap : float
         Emission per capacity
     """
-    emission_cap = (clim_emissions / monthly_traffic) * 12 * 5
+    emission_cap = (clim_emissions / monthly_traffic) / (12 * 5)
```

Comment:
What is 5? This should be in the parameters dictionary with a proper key e.g, "time_horizon". Please make sure all parameters are in the parameters dictionary.

Reply:
I will accept this for now, but we still need to use a parameters dictionary more thoroughly in the codebase, as it helps you and the reviewers have all the key model inputs in one place.